<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDs3</class>
 <widget class="QMainWindow" name="IDs3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>631</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>631</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 3 TID\SID</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">.QTabWidget &gt;  QWidget &gt; QWidget {
	background-color: #E6E6E6;
}

QTabBar::tab {
	color: black;
	background-color: #A1ABB3;
	font-family: &quot;Myriad Pro&quot;;
}

QTabBar::tab:selected {
	color: white;
	background-color: #1D303C;
}

QCheckBox {
	color: white;
}

QCheckBox::indicator {
	border: 2px solid white;
	border-radius: 3px;
	height: 10px;
	width: 10px;
}

QCheckBox::indicator:unchecked {
	background-color: #274050;
}

QCheckBox::indicator:checked {
	background-color: white;
}

QTableView {
	border: 4px solid DodgerBlue;
}

QHeaderView::section {
	border: none;
	background-color: #274050;
	padding-top: 3px;
	padding-bottom: 3px;
	color: white;
}

QLineEdit {
	qproperty-alignment: AlignCenter;
	background-color: #1D303C;
	border:none;
	border-radius: 6px;
	color: white;
}

QGroupBox {
	background-color: #274050;
	border-radius: 10px;
	margin-top: 7px;
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

.QGroupBox * {
	font-family: &quot;Myriad Pro&quot;, Montserrat;
	font-weight: 500;
}

QLabel {
	color: white;
}

QGroupBox::title {
	subcontrol-origin: margin;
	left: 20px;
	padding: 3px 5px 3px 5px;
	background-color: #969F91;
	border-radius: 5px;
}

QPushButton {
	border: none;
	border-radius: 6px;
	color: white;
}

QRadioButton::indicator {
	border: 2px solid white;
	border-radius: 3px;
	height: 10px;
	width: 10px;
}

QRadioButton::indicator:checked {
	background-color: white;
}

QRadioButton::indicator:unchecked {
	background-color: #274050;
}

QRadioButton {
	color: white;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>XD/Colo</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxSearchBy_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Search By</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxPIDXD">
             <property name="text">
              <string>PID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextBox" name="textBoxPIDXD">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxTIDXD">
             <property name="text">
              <string>TID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxTIDXD">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBoxSIDXD">
             <property name="text">
              <string>SID</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextBox" name="textBoxSIDXD">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButtonFindXD">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxParameters_3">
          <property name="title">
           <string>Search Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelPRNG">
             <property name="text">
              <string>Seed</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextBox" name="textBoxSeedXD">
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMinFrame_3">
             <property name="text">
              <string>Starting Frame</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxMinFrameXD">
             <property name="text">
              <string notr="true">5000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelMaxFrame_3">
             <property name="text">
              <string>Max Results</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextBox" name="textBoxMaxFrameXD">
             <property name="text">
              <string notr="true">10000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TableView" name="tableViewXDColo">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gen3FRLGE">
       <attribute name="title">
        <string>Gen III FRLGE</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxSearchBy">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Search By</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxPIDFRLGE">
             <property name="text">
              <string>PID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextBox" name="textBoxPIDFRLGE">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxSIDFRLGE">
             <property name="text">
              <string>SID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxSIDFRLGE">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButtonFindFRLGE">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxParameters">
          <property name="title">
           <string>Search Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTID">
             <property name="text">
              <string>TID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextBox" name="textBoxTIDFRLGE">
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelMinFrame">
             <property name="text">
              <string>Starting Frame</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxMinFrameFRLGE">
             <property name="text">
              <string notr="true">5000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelMaxFrame">
             <property name="text">
              <string>Max Results</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextBox" name="textBoxMaxFrameFRLGE">
             <property name="text">
              <string notr="true">10000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TableView" name="tableViewFRLGE">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gen3RS">
       <attribute name="title">
        <string>Gen III RS</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBoxSearchBy_2">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="title">
           <string>Search By</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxPIDRS">
             <property name="text">
              <string>PID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextBox" name="textBoxPIDRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>8</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxTIDRS">
             <property name="text">
              <string>TID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextBox" name="textBoxTIDRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBoxSIDRS">
             <property name="text">
              <string>SID</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextBox" name="textBoxSIDRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxLength">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QPushButton" name="pushButtonFindRS">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>23</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: DodgerBlue;
}</string>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxParameters_2">
          <property name="title">
           <string>Search Parameters</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="1" column="3">
            <widget class="QRadioButton" name="radioButtonInitSeedRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Initial Seed</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxBattery">
             <property name="text">
              <string>Dead Battery</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonDateRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Date</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QDateTimeEdit" name="dateTimeEdit">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="dateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="maximumDateTime">
              <datetime>
               <hour>23</hour>
               <minute>59</minute>
               <second>59</second>
               <year>2037</year>
               <month>12</month>
               <day>31</day>
              </datetime>
             </property>
             <property name="minimumDateTime">
              <datetime>
               <hour>0</hour>
               <minute>0</minute>
               <second>0</second>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </datetime>
             </property>
             <property name="minimumDate">
              <date>
               <year>2000</year>
               <month>1</month>
               <day>1</day>
              </date>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QTextBox" name="textBoxInitSeedRS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>4</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QTextBox" name="textBoxMinFrameRS">
             <property name="text">
              <string notr="true">5000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QTextBox" name="textBoxMaxFrameRS">
             <property name="text">
              <string notr="true">10000</string>
             </property>
             <property name="maxLength">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="labelMinFrame_2">
             <property name="text">
              <string>Starting Frame</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="labelMaxFrame_2">
             <property name="text">
              <string>Max Results</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="TableView" name="tableViewRS">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/QTextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
